server/config/supabase.js

```diff
@@ -2,5 +2,7 @@ const { createClient } = require("@supabase/supabase-js");
 
 const supabaseUrl = process.env.SUPABASE_PROJECT_URL;
 const supabaseKey = process.env.SUPABASE_ANON_KEY;
+const SECRET_KEY = process.env.SECRET_KEY;
```

Comment:
I think we could change this line to `const SECRET_KEY = process.env.SUPABASE_SECRET;` just because `SUPABASE_SECRET` is name people will have in their .envs from the original file that was shared in discord.

Reply:
done, just updated it.